archive_upload/handlers/dsmc_handlers.py

```diff
@@ -468,6 +469,12 @@ def post(self, runfolder_archive):
         log.info("Uploading {} to PDC...".format(path_to_archive))
         cmd = "export DSM_LOG={} && dsmc archive {}/ -subdir=yes -description={}".format(
             dsmc_log_dir, path_to_archive, uniq_id)
+
+        # Mock starting the TSM process if mock mode is enabled
+        tsm_mock_enabled = self.config.__getitem__("tsm_mock_enabled")
+        if tsm_mock_enabled:
+            self.runner_service.start = Mock(return_value=self.config["tsm_mock_job_id"])
```

Comment:
I think we should throw in a log warning here, if we should ever happen to turn it on in production we have evidence of it.

Reply:
Done!